zwave_js/CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## 0.8.0
+
+### Features
+
+- Add-on: Add `disable_watchdog` configuration option. When enabled, the driver will not enable the hardware watchdog of the Z-Wave controller. This is an advanced configuration option that should not be adjusted in most cases and is therefore hidden from the default view.
+- Z-Wave JS: Multiple parallel firmware updates are now supported

```

Comment:
Not sure if this is possible right now in HA. At least `node-zwave-js` supports it.

Reply:
What does it mean?

Reply:
Before, zwave-js would error when one firmware update was going on (or just queued) and the user would try to start another one. Now this is possible.

I'm not sure though if the integration or addon has an additional check on top.

Reply:
Was this limitation only per device or for the whole driver?

Reply:
For the whole driver.

Reply:
Ok. I'm not aware of any such limitation in Home Assistant.